types/daterangepicker/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for Date Range Picker v2.1.25
+// Type definitions for Date Range Picker v2.1.30
```

Comment:
The latest release is 2.1.27

Reply:
It's confusing because there are two separate npm packages with the same library:

https://www.npmjs.com/package/bootstrap-daterangepicker is at 2.1.30
https://www.npmjs.com/package/daterangepicker is at 2.1.27

Both have more-or-less identical code.

Reply:
Both links point to the same github repo and under releases there is only a `2.1.27`. From what I can tell `2.1.30` is only the npm package version number not the git tag. Daft though :)